account_multicompany_code/views/res_company_views.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_company_form" model="ir.ui.view">
+        <field name="name">qf_common.company.form</field>
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="base.view_company_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_id']" position="before">
+                <field name="code" required="1" />
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_company_tree" model="ir.ui.view">
+        <field name="name">qf_common.company.tree</field>
```

Comment:
Fix name, I'd recommend:

`res.company.tree.inherit.account.multicompany`

I just noticed the existing view was also wrong, so please fix also that one